tests/src/test/java/stepDefinitions/frontendstepdefinitions/EFileSubmissionTest.java

```diff
@@ -0,0 +1,179 @@
+package stepDefinitions.frontendstepdefinitions;
+
+import ca.bc.gov.open.jagefilingapi.qa.config.ReadConfig;
+import ca.bc.gov.open.jagefilingapi.qa.frontend.pages.EFileSubmissionPage;
+import ca.bc.gov.open.jagefilingapi.qa.frontend.pages.LandingPage;
+import ca.bc.gov.open.jagefilingapi.qa.frontendutils.DriverClass;
+import ca.bc.gov.open.jagefilingapi.qa.frontendutils.JsonDataReader;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import java.io.IOException;
+
+public class EFileSubmissionTest extends DriverClass {
+
+     ReadConfig readConfig;
+     LandingPage landingPage;
+     EFileSubmissionPage eFileSubmissionPage;
+     private static final String EFILE_SUBMISSION_PAGE_TITLE = "eFiling Frontend";
```

Comment:
title should be `E-File Submission` if you want to match what is on header on frontend app 👍

Reply:
Thanks @sdevalapurkar-bcgov. This string constant is to assert based on the retrieved page title to confirm user is on the right page. Will consider modifying if this is unique for this page when we have other pages. Thanks!